src/main/java/com/internet/shop/util/DbConnectionUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.internet.shop.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DbConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+        } catch (ClassNotFoundException e) {
+            throw new RuntimeException("Can't find MySQL Driver", e);
+        }
+    }
+
+    public static Connection getConnection() throws SQLException {
+        Properties connectionProps = new Properties();
+        connectionProps.put("user", "root");
+        connectionProps.put("password", "Vi4ft5q7Ktrr9T6toPcN");
```

Comment:
do not push your real credentials to GutHub. Never

Reply:
Ok. Done